hardware/Layout-Desing-Guide.md

```diff
@@ -0,0 +1,486 @@
+    - [3.1. Layout Design Practices - 2.4 GHz matching network](#3.1.-Layout-Design-Practices---2.4-GHz-matching-network)
+    - [3.2. Layout Design Practices - Sub-GHz matching network](#3.2.-Layout-Design-Practices---Sub-GHz-matching-network)
+    - [3.3. Layout Design Practices - HFXO](#3.3.-Layout-Design-Practices---HFXO)
+    - [3.4. Layout Design Practices - VDD filtering](#3.4.-Layout-Design-Practices---VDD-filtering)
+    - [3.5  Layout Design Practices - General](#3.5.-Layout-Design-Practices---General)
+  - [4. Quiz](#4.-Quiz)
+
+</details>
+
+
+
+
+The purpose of this material is to help users design PCBs
+for the EFR32 Wireless Gecko Portfolio using design practices
+that allow for good RF performance.
+
+PCB layout requirements are described in the application note, [AN928.1: EFR32 Series 1 Layout Design Guide](https://www.silabs.com/documents/public/application-notes/an928.1-efr32-series1-layout-design-guide.pdf),  and the layout design process for EFR32 Series 1 Modules is discussed in [the datasheet of each module](https://www.silabs.com/support/resources.ct-data-sheets.ct-miscellaneous.p-wireless_bluetooth-low-energy_blue-gecko-bluetooth-low-energy-modules.p-wireless_bluetooth-low-energy_blue-gecko-bluetooth-low-energy-modules_bgm11s12f256ga-v2.p-wireless_bluetooth-low-energy_efr32bg1-series-1-modules.p-wireless_bluetooth-low-energy_efr32bg13-series-1-modules.sort=2,asc) and  besides that EFR32 reference design files are available in Simplicity Studio.
+
+
+# 1. Why is the proper layout design important? 
+
+If the layout is an inappropriate design that can cause 
++ Degradation in the communication distance 
++ The unit can violate the relevant EMC regulation (high harmonics, spur) 
++ The battery life can be reduced
+
+# 2. RF section 
+
+
+## 2.1. Different parts of RF section
+The most important part of the proper layout design is the RF circuitry itself.
+
+In the case of dual band RFIC, this RF part includes the following:
+1. Sub-GHz and 2.4 GHz RF matching newtork  
+2. Power supply filterings 
+3. High frequency crystal  
+
+
+
+<p align="center">
+  "<img src="files/Layout-Design-Guide/matching network.png">
+</p>
+
+<div align="center">
+
+<b>Figure 1. RF section</b> 
+
+</div>
+
+     
+
+
+### 2.1.1. Function of RF matching  and filtering network
+
+The RF transmitter ICs can synthetize high frequency and high power signals at their TX output: the fundamental, its harmonics and additional spurious content.
+On the Silicon Labs’ reference design boards the function of the TX matching is to drive most of the fundamental power into the antenna, thus to provide the required impedance transformation between the RFIC and the load. Besides that, a proper filtering network should be used between the matching network and the antenna design which is able to suppress harmonics and spurs. RX matching network is able to provide impedancia tuning between IC and the load. 
```

Comment:
Change "Silicon Labs' reference" to "Silicon Labs reference".
Change "impedancia" to "impedance".

Reply:
:) it was in Hungarian